src/components/events/CalendarButtons/style.module.scss

```diff
@@ -0,0 +1,35 @@
+.options {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.calendarButton {
+  align-items: center;
+  background: transparent;
+  border: 1px solid var(--theme-primary-2);
+  border-radius: 10px;
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+  padding: 0.75rem;
+  width: 15rem;
+}
+
+.calendarButton:hover {
+  background: var(--theme-primary-2);
+  transition: all 0.2s;
```

Comment:
you should move this `transition` property to `.calendarButton` itself. this way, there's also an animation when your mouse moves off the button

Also, you could move this rule inside `.calendarButton` like this:

```scss
.calendarButton {
  ...
  transition: all 0.2s;
  
  &:hover {
    background: var(--theme-primary-2);
  }
}
```

This has the same result, but it makes it more clear that these two rules are related

Reply:
good catch! fixed